db/upgrade-X.X.X-X.Y.Z.sql

```diff
@@ -1,3 +1,33 @@
 --
 -- PacketFence SQL schema upgrade from X.X.X to X.Y.Z
 --
+
+--
+-- Add a primary key to the radacct_log
+--
+
+ALTER TABLE `radacct_log` ADD COLUMN `radacct_log_id` int NOT NULL PRIMARY KEY AUTO_INCREMENT FIRST;
```

Comment:
Since there is no real standard I choose the most explicit naming convention.

So there is no confusion on what that field is for.

Also it allow to write joins using 'USING(tablename_id)' syntax.